app/controllers/projects_controller.rb

```diff
@@ -16,12 +17,23 @@ def show
     @authors       = [current_user]
     @boards        = current_project.methodology_library.boards
     @issues        = current_project.issues.includes(:tags).sort
-    @methodologies = current_project.methodology_library.notes.map{|n| Methodology.new(filename: n.id, content: n.text)}
+    @methodologies = current_project.methodology_library.notes.map { |n| Methodology.new(filename: n.id, content: n.text) }
     @nodes         = current_project.nodes.in_tree
     @tags          = current_project.tags
 
+    project_cards = @boards.map(&:cards)[0]
```

Comment:
This will only pick up the cards from the first board in the project. Instead we can move this to a helper and do:

L23: `@tasks = assigned_tasks`
```
def assigned_tasks
  @assigned_tasks ||= begin
    cards = @boards.flat_map(&:cards)
    user_cards = cards.select { |card| card.assignees.include? current_user }
    user_cards.sort_by { |card| card.due_date || Float::INFINITY }
  end
end
```